manifests/descheduler-operator-cr.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: cluster
   namespace: openshift-kube-descheduler-operator
 spec:
+  mode: Automatic
   managementState: Managed
-  devEnableEvictionsInBackground: true
```

Comment:
I think we have to keep this, no?

Reply:
Not here, it's already correctly on line 15 under `profileCustomizations`

Reply:
10x!